### DISCUSSION TITLE
How to configure the default fields?

### DISCUSSION THREAD

**Discussion by justinphelps** [ankitpokhrel/jira-cli#257] [Q&A]
By default, a list query returns these columns:
TYPE KEY SUMMARY STATUS ASSIGNEE REPORTER PRIORITY RESOLUTION CREATED UPDATED

Is it possible to configure a different selection of default columns? e.g., By default I only want to see KEY, UPDATED without having to use --columns every time.

**Comment by ankitpokhrel** [comment]
Hi @justinphelps, there isn't a way to configure default fields at the moment. I will consider making it configurable if there is a demand or if someone sends a PR :). For now, you can create an alias for frequently used commands.
